test/system/admin/admin_auctions_test.rb

```diff
@@ -113,7 +113,7 @@ def test_administrator_can_see_all_created_auctions
     assert(page.has_link?('expired.test', href: admin_auction_path(@expired_auction.id)))
 
     prices = page.find_all('.auction-highest-offer').map(&:text)
-    assert_equal(['50.00', '10.00', '0.00', '100.00'].to_set, prices.to_set)
+    assert_equal(['50.00', '10.00', I18n.t('nan'), '100.00'].to_set, prices.to_set)
```

Comment:
Perhaps it's better to hardcode this? If it's not translated, `nil` will be returned, and I am not sure how this will behave then.
I wonder whether it's really a system test. Can this logic be tested in unit tests?

Reply:
I suppose it's a system test indeed - we might want to see if view behaves as expected and shows 'NaN' if there is no data.
But hardcode is a thing thought, made it so.